planner/core/physical_plans.go

```diff
@@ -485,6 +485,9 @@ type PhysicalSort struct {
 // appear in final physical operator tree.
```

Comment:
Also, add comments here to record that `NominalSort` will be eliminated or converted to `Projection`.